artifacts/attributes.adoc

```diff
@@ -13,7 +13,7 @@
 :product-version: 1.4
 :product-bundle-version: 1.4.0
 :product-chart-version: 1.4.0
-:product-backstage-version: 1.29.2
+:product-backstage-version: 1.31.3
```

Comment:
for RHDH 1.4 this should be `1.32.6`
source https://github.com/janus-idp/backstage-showcase/blob/release-1.4/backstage.json